client/src/Pages/Home.js

```diff
@@ -27,7 +34,7 @@ const Home = () => {
             <h5 className="card-title">{post.dish} from {post.restaurant}</h5>
             <p className="card-text-single">{post.content}</p>
             
-            <a href="#" className="btn btn-primary">
+            <a onClick={checkShit} href="#" className="btn btn-primary">
```

Comment:
Good idea to make this a button. Anchor tag is a pain to get it to behave as a button.

Reply:
debating the button all together on the homepage view